### PR TITLE
Test dimensions followup

### PR DESCRIPTION
**Pull request by keyonghan**
This is a follow PR of https://github.com/flutter/cocoon/pull/1597 to move `getDimensions` function to Target.

### CODE REVIEW

File: app_dart/lib/src/service/config.dart [hunk 1]
```diff
@@ -97,7 +97,7 @@ class Config {
   }
 
   // Dimension list defined in .ci.yaml.
-  List<String> get dimensionList => <String>['os', 'device_os', 'device_type', 'mac_model'];
+  static List<String> get dimensionList => <String>['os', 'device_os', 'device_type', 'mac_model'];
```
Comment (CaseyHillers) [inline app_dart/lib/src/service/config.dart:100]: Should this be moved into `Target`? I can't see a reason why these need to be accessible outside of `Target`.

File: app_dart/lib/src/model/ci_yaml/target.dart [hunk 2]
```diff
@@ -33,6 +35,18 @@ class Target {
   /// Target prefixes that indicate it will run on an ios device.
   static const List<String> iosPlatforms = <String>['mac_ios', 'mac_ios32'];
 
+  /// Gets dimensions for this [pb.Target].
```
Comment (CaseyHillers) [inline app_dart/lib/src/model/ci_yaml/target.dart:38]: Can you expand on what dimensions are? Possibly a link to the swarming documentation

**Review by CaseyHillers — COMMENTED** [review]
Thank you for the quick follow up!

File: app_dart/lib/src/service/config.dart (hunk 1 above)
Comment (keyonghan) [reply app_dart/lib/src/service/config.dart]: Done.

**Review by CaseyHillers — APPROVED** [review]
flutter-lgtm

**Comment by CaseyHillers** [comment]
FYI tests are failing due to formatting issues

**Pull request merged**